packages/web3/src/index.ts

```diff
@@ -327,3 +327,8 @@ export * from './web3';
 export * from './types';
 
 export default Web3;
+
+// The following JavaScript command is used instead of the equivalent TypeScript `export = Web3`, 
+//  to allow exporting the types along with Web3 constructor as the default function,
+//  in order to maintain backward compatibility.
+module.exports = Web3;
```

Comment:
Need to disable the TS error. 

```suggestion
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-expect-error
module.exports = Web3;
```

Reply:
Thanks for the hint. However, I changed the approach :blush: